src/Database/Bloodhound/Client.hs

```diff
@@ -995,12 +995,21 @@ encodeBulkOperation (BulkUpdate (IndexName indexName)
 encodeBulkOperation (BulkUpsert (IndexName indexName)
```

Comment:
Are we using this function externally? The changes have altered the API.

Reply:
We do _construct_ `BulkUpsert` externally, yes. 
It is an application code, and that code will have to change.

Reply:
Fair enough. I doubt we were using the changed bits, anyway, and we're going to touch all those projects.